packages/docusaurus-theme-search-algolia/src/validateThemeConfig.ts

```diff
@@ -39,6 +39,10 @@ export const Schema = Joi.object<ThemeConfig>({
       .try(Joi.boolean().invalid(true), Joi.string())
       .allow(null)
       .default(DEFAULT_CONFIG.searchPagePath),
+    replaceInItemUrl: Joi.object({
```

Comment:
not super fan of `replaceInItemUrl` as a name, but I don't have much better 😅 

What about `replaceSearchResultPathname` ?